fairmq/Properties.cxx

```diff
@@ -8,7 +8,15 @@
 
 #include <fairmq/Properties.h>
 
+#if FAIRMQ_HAS_STD_FILESYSTEM
+#include <filesystem>
+namespace fs = std::filesystem;
+#else
+#define BOOST_FILESYSTEM_VERSION 3
+#define BOOST_FILESYSTEM_NO_DEPRECATED
 #include <boost/filesystem.hpp>
+namespace fs = ::boost::filesystem;
```

Comment:
I wonder how breaking this is. 
If some use has a program option using boost::filesystem::path.

Couldn't find any users by searching atm.

Reply:
It will be breaking in general, but user can set `-DFAIRMQ_HAS_STD_FILESYSTEM=0` to switch back to boost and adapt at her own schedule.

Reply:
And the `fs` alias should not break anything.